SQLQuery_2509.sql

```diff
@@ -0,0 +1,57 @@
+--SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_TYPE='BASE TABLE';
+
+--CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+CREATE TABLE Employee(EmployeeID int primary key, FirstName varchar(20), LastName varchar(20),Department varchar(5),Salary int);
```

Comment:
Add necessary constraints, why use int for salary

Reply:
Noted and corrected